local-cli/server/middleware/getDevToolsMiddleware.js

```diff
@@ -24,7 +25,39 @@ function getChromeAppName() {
   }
 }
 
-module.exports = function(options, isDebuggerConnected) {
+function launchChromeDevTools(port) {
+  var debuggerURL = 'http://localhost:' + port + '/debugger-ui';
+  console.log('Launching Dev Tools...');
+  opn(debuggerURL, {app: [getChromeAppName()]}, function(err) {
+    if (err) {
+      console.error('Google Chrome exited with error:', err);
+    }
+  });
+}
+
+function escapePath(path) {
+  return '"' + path + '"'; // " Can escape paths with spaces in OS X, Windows, and *nix
```

Comment:
Escaping path components is so much more complex than this, please use `child_process.spawn` instead of `exec`, it should handle the escaping correctly.